src/components/Sponsors/Sponsors.tsx

```diff
@@ -3,12 +3,15 @@ import { useColorMode } from '@docusaurus/theme-common';
 import { sanitize } from 'dompurify';
 
 export default function Sponsors(): React.JSX.Element {
-  const [data, setData] = useState(null);
   const { colorMode } = useColorMode();
+  const [data, setData] = useState<string | null>(localStorage.getItem(colorMode));
```

Comment:
```suggestion
  const [data, setData] = useState <string | null> (localStorage.getItem(colorMode));
```

Reply:
![image](https://github.com/betaflight/betaflight.com/assets/8344830/f7e9e1bd-3d1b-4cf3-82a9-8bc65daac9e3)

Reply:
We only have a javascript label. Does it even matter or do we need to introduce a typescript label?

Reply:
Added new label.